src/main/scala/org/broadinstitute/dsde/rawls/model/VaultModel.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.rawls.model
```

Comment:
The current convention is that all model classes go in ```WorkspaceModel.scala```. Although that file is getting pretty bloated, so maybe we should break it up.